wakeup-server/tests/test_interactive_devices_exception.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+
+def test_empty_name(client):
+    response = client.post("/interactive_devices", json={"name": "", "type": "SOUND"})
+    obj = response.get_json()
+    with pytest.raises(TypeError) as e:
+        name = obj['name']
+    msg = e.value.args[0]
+    assert 'NoneType' in msg
+
+
+def test_too_long_name(client):
+    response = client.post("/interactive_devices", json={"name": "a"*1000, "type": "SOUND"})
+    obj = response.get_json()
+    with pytest.raises(Error) as e:
+        name = obj['name']
+        assert name == "a"*1000
+
+
+def test_duplicate_name(client):
+    response = client.post("/interactive_devices", json={"name": "test_1", "type": "SOUND"})
+    with pytest.raises(IntegrityError) as e:
+        response2 = client.post("/interactive_devices", json={"name": "test_1", "type": "SOUND"})
```

Comment:
Why a second post request ?

Reply:
create a new device with same name and type to see if the server allows duplicate

Reply:
Ah make sense ! Sorry my bad